### ISSUE TITLE
how to understand the loss of shape and expression ? 

### ISSUE THREAD

**Issue by CaichaoGitHub** [YadiraF/DECA#70]
in paper ,
 it says the shape loss will be calculated between two some person , but what's the ground truth ?
at same time , i don't see any description of the expression loss ;

the blew is the training code :
`losses['shape_reg'] = (torch.sum(codedict['shape']**2)/2)*self.cfg.loss.reg_shape`
`losses['expression_reg'] = (torch.sum(codedict['exp']**2)/2)*self.cfg.loss.reg_exp`

thanks for sharing
